libs/hmpb/src/ballot_fixtures.ts

```diff
@@ -72,55 +87,113 @@ export const vxFamousNamesFixtures = (() => {
     electionDefinition,
     blankBallotPath,
     markedBallotPath,
+    blankOfficialBallotPath,
+    markedOfficialBallotPath,
     allBallotProps,
+    allBallotPropsTest,
+    allBallotPropsOfficial,
     ...blankBallotProps,
     votes,
 
     async generate(
       rendererPool: RendererPool,
       { generatePageImages = false } = {}
     ) {
+      // Generate test mode ballots
       debug(`Generating: ${blankBallotPath}`);
-      const layouts = await layOutBallotsAndCreateElectionDefinition(
+      const layoutsTest = await layOutBallotsAndCreateElectionDefinition(
         rendererPool,
         vxDefaultBallotTemplate,
-        allBallotProps,
+        allBallotPropsTest,
         'vxf'
       );
 
       assert(
-        layouts.electionDefinition.ballotHash === electionDefinition.ballotHash,
+        layoutsTest.electionDefinition.ballotHash ===
+          electionDefinition.ballotHash,
         'If this fails its likely because the lib/fixtures election fixtures are out of date. Run pnpm generate-election-packages in libs/fixture-generators'
       );
 
-      const blankBallotContents = layouts.ballotContents[0];
-      const { blankBallotPdf, markedBallotPdf } = await rendererPool.runTask(
-        async (renderer) => {
-          const ballotDocument =
-            await renderer.loadDocumentFromContent(blankBallotContents);
-          // eslint-disable-next-line @typescript-eslint/no-shadow
-          const blankBallotPdf = await renderBallotPdfWithMetadataQrCode(
-            allBallotProps[0],
-            ballotDocument,
-            electionDefinition
-          );
-
-          debug(`Generating: ${markedBallotPath}`);
-          await markBallotDocument(ballotDocument, votes);
-          // eslint-disable-next-line @typescript-eslint/no-shadow
-          const markedBallotPdf = await ballotDocument.renderToPdf();
+      // Generate official mode ballots
+      debug(`Generating: ${blankOfficialBallotPath}`);
+      const layoutsOfficial = await layOutBallotsAndCreateElectionDefinition(
+        rendererPool,
+        vxDefaultBallotTemplate,
+        allBallotPropsOfficial,
+        'vxf'
+      );
 
-          return { blankBallotPdf, markedBallotPdf };
-        }
+      assert(
+        layoutsOfficial.electionDefinition.ballotHash ===
+          electionDefinition.ballotHash,
+        'If this fails its likely because the lib/fixtures election fixtures are out of date. Run pnpm generate-election-packages in libs/fixture-generators'
       );
 
+      const blankBallotContents = layoutsTest.ballotContents[0];
```

Comment:
I was gonna suggest renaming this `blankTestBallotContents` and making a similar change for other variables in this pipeline, e.g., `blankTestBallotPdf`, `markedTestBallotPdf`, blankTestBallotPageImages`, etc., but I see that you might have to rename quite a lot. Okay leaving as is if renaming all the way through is gonna be too cumbersome